docs/walkthrough.md

```diff
@@ -175,7 +175,7 @@ spec:
 If you try creating that now (and take a look at your controller-manager
 logs), you'll see that the that the HorizontalPodAutoscaler controller is
 attempting to fetch metrics from
-`/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/pods/*/http_requests?selector=app%3Dsample-app`,
+`/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/pods/*/http_requests?metricLabelSelector=app%3Dsample-app`,
```

Comment:
The selector should be `labelSelector` instead of `metricLabelSelector` since `app=sample-app` is a label of the pod, not of the metric.

Reply:
Ok, nevermind